xrdp/xrdp.h

```diff
@@ -411,6 +411,9 @@ get_char_from_scan_code(int device_flags, int scan_code, int *keys,
 int
 get_keymaps(int keylayout, struct xrdp_keymap *keymap);
 
+int
+km_load_file(const char *filename, struct xrdp_keymap *keymap);
+
```

Comment:
I notice this is a global. Is the intention to use it elsewhere later on?

Reply:
Yes, this is intentional and I'm going to add some unit tests later.